proposal/iblis17/README.rst

```diff
@@ -0,0 +1,53 @@
+===============================================================================
+Proposal for Heterogeneously-typed and Dynamic Time Series Data Structure of TimeSeries.jl
+===============================================================================
+
+
+Basic information
+=================
+
```

Comment:
You need to add a repository to your project.  You may use an existing one, but need to be able to make it clear what is your contribution.

Reply:
https://github.com/JuliaStats/TimeSeries.jl seems to be your upstream.  If you intend to directly working in the upstream, please give more details about how it will be done in the proposal.

Reply:
Ah, okay, I added a link explicitly.

Reply:
> https://github.com/JuliaStats/TimeSeries.jl seems to be your upstream. If you intend to directly working in the upstream, please give more details about how it will be done in the proposal.

well, just send a PR and commit to that repo. I will create a branch on it, but that's kind of trivial detail that I don't want to mention it in this proposal.

Reply:
No problem.  Mention it here in the PR is good enough, after you explain it.